edb/edgeql/compiler/casts.py

```diff
@@ -97,15 +97,14 @@ def compile_cast(
             f'`...[IS {new_stype.get_displayname(ctx.env.schema)}]` instead',
             context=srcctx)
 
-    uuid_t = ctx.env.get_schema_type_and_track(sn.QualName('std', 'uuid'))
```

Comment:
I don't like the `"` quote thing at all; could you redo the darker step with that turned off? (IIRC that is like one of two config flags that black has.)